codex-cli/src/utils/providers.ts

```diff
@@ -26,6 +26,7 @@ export const providers: Record<
     name: "Ollama",
     baseURL: "http://localhost:11434/v1",
     envKey: "OLLAMA_API_KEY",
+    noApiKeyRequired: true,
```

Comment:
Is there ever a case where `OLLAMA_API_KEY` makes sense? I wasn't sure that there was.

If that's the case, can we just make `envKey` optional instead of adding `noApiKeyRequired`?

If we do need to make it optional, I think `envKeyIsOptional` or something with a "positive" sense would be better than something in the "negative" sense that starts with "no."

Reply:
From my understanding, [some people seem to want it](https://github.com/ollama/ollama/pull/9131), but there is no official support, and it would be optional if added. So your proposal makes sense, I'll remove the requirement for `envKey` instead. I'll keep the old "Ollama" hardcoded values for backwards compatibility.